ports/quake.2/Quake 2 - Capture The Flag.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+source $controlfolder/device_info.txt
+
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+get_controls
+
+to_lower_case() {
+    find "$1" -depth \( -name "*.PAK" -o -name "PAK0.pak" \) | while IFS= read -r SRC; do
+        DST=$(dirname "$SRC")/$(basename "$SRC" | tr '[:upper:]' '[:lower:]')
+        TMP_DST=$(dirname "$SRC")/temp_$(basename "$SRC" | tr '[:upper:]' '[:lower:]')
+        echo "SRC: $SRC"
+        echo "DST: $DST"
+        echo "TMP_DST: $TMP_DST"
+        if [ "$SRC" != "$DST" ]; then
+            echo "Renaming $SRC to $TMP_DST"
+            $ESUDO mv -vf "$SRC" "$TMP_DST"
+            echo "Renaming $TMP_DST to $DST"
+            $ESUDO mv -vf "$TMP_DST" "$DST"
+        else
+            echo "- $SRC is already lowercase"
+        fi
+    done
+}
+
+#GAMEDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )/quake2"
+GAMEDIR="/$directory/ports/quake2"
+
+$ESUDO chmod 777 -R $GAMEDIR/*
+
+cd $GAMEDIR
+
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+to_lower_case "$GAMEDIR/ctf"
+
+$ESUDO chmod 666 /dev/tty0
+$ESUDO chmod 666 /dev/tty1
+$ESUDO chmod 666 /dev/uinput
+
+export SDL_GAMECONTROLLERCONFIG="$sdl_controllerconfig"
+
+if [[ "$DISPLAY_WIDTH" == '480' ]]; then
+    SSCALE="1"
+else
+    SSCALE="3"
+fi
+
+if [[ "${CFW_NAME^^}" == 'JELOS' ]] || [[ "${CFW_NAME^^}" == 'ROCKNIX' ]]; then
```

Comment:
Does this apply to mali aswell because they have maali and panfrost.

Reply:
With Ganimoth's final testing and confirming that he had issues with GLES3 with ROCKNIX, I'm going to leave it to start in software mode for these two CFWs.  FWIW, this won't be a downgrade from what we already have for Quake II and will continue to work using software rendering on these devices.  The user can change it in the video settings if they wish to try it and/or alter the .sh to override the software renderer with gles3 if they so desire.  This will still be an upgrade for ArkOS, AmberELEC, muOS, and several other CFWs as the GLES3 API works on almost all other firmwares without issue.